pfcp/handler/handler.go

```diff
@@ -157,6 +167,12 @@ func HandlePfcpAssociationSetupRequest(msg *udp.Message) {
 
 	nodeID := smf_context.NewNodeID(nodeIDStr)
 
```

Comment:
Did you test incoming PFCP Association Setup Request?

Reply:
I've tested the whole end-to-end flow, which includes the PFCP association setup request and it worked fine. Let me know if something is missing here.